proto/osmosis/superfluid/tx.proto

```diff
@@ -150,8 +150,8 @@ message MsgUnlockAndMigrateSharesToFullRangeConcentratedPosition {
   ];
   // token_out_mins indicates minimum token to exit Balancer pool with.
   repeated cosmos.base.v1beta1.Coin token_out_mins = 4 [
-    (gogoproto.moretags) = "yaml:\"token_out_min_amounts\"",
-    (gogoproto.nullable) = false
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
```

Comment:
I think we need to keep gogo proto moretags here for front end?

Reply:
Added back proto tag here https://github.com/osmosis-labs/osmosis/pull/5178/commits/5c82ca3ee9001e9aa00eb0cdc13f580f48593f83